src/EditTodoItem.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import './EditTodoItem.css'
+
+class EditTodoItem extends React.Component {
+  
+  constructor(props) {
+    super(props)
```

Comment:
Please configure linting in this repo

Reply:
Can you give a reference of which linting should we use for React? @mashhoodr